libsolidity/codegen/YulUtilFunctions.h

```diff
@@ -72,6 +72,15 @@ class YulUtilFunctions
 	/// Pads with zeros and might write more than exactly length.
 	std::string copyToMemoryFunction(bool _fromCalldata);
 
+	/// @returns a function name that copies string literal to memory
+	/// and returns pointer to memory array containing it
+	/// signature: () -> memPtr
+	std::string copyLiteralToMemoryFunction(std::string const& _literal);
+
+	/// @returns a function name that stores string literal to specific location in memory
+	/// signature: (memPtr) ->
+	std::string storeLiteralToMemoryFunction(std::string const& _literal);
```

Comment:
A bit pedantic, but `store ... to` sounds a bit weird to me, I'd replace it to `storeLiteralInMemoryFunction`

Reply:
Changed